### ISSUE TITLE
no way to download a non transcoded file

### ISSUE THREAD

**Issue by C2gl** [fredrikburmester/streamyfin#115]
when downloading anything on IOS, there seems to be no option to download the original file without having the server transcode it, i also tried enabling native mode, but that doesn't seem to solve the issue 

i tried this on both H265, H264, AV1 files, all automatically transcode 

i tried this both on an ipad 2018, and an iphone 15, both running the latest available IOS (17.6.1)

**Comment by fredrikburmester** [comment]
This is true. This is because if you download a file that is not in a supported format/codec, it wont play. Therefore i automatically choose the download strategy, either direct stream or transcoded. 

Choosing "Max" (or the highest bitrate) will download the original file **if it is in a format that your phone supports.** 

The reason for transcoding might not be the video file (but it can be), rather the audio or the subtitles.

**Comment by Shah-do** [comment]
For me (on android) the app seems to be saying remuxing to Mp4 even though my phone does support MKV (streaming it even plays it directly without any change) why is it so?
